diff --git a/src/validator/index.js b/src/validator/index.js
index a9b7a013ad67396ae805a8467d9799b57d52c51c..152149dbcf4b10b83095bd1e6462b8830e1ae498 100644
--- a/src/validator/index.js
+++ b/src/validator/index.js
@@ -98,11 +98,7 @@ export class Validator {
       }
 
       try {
-        const parsed = new URL(addHttps(value))
-
-        if (!parsed.hostname.includes('.')) {
-          return message
-        }
+        new URL(addHttps(value))
 
         return null
       } catch {
diff --git a/src/validator/index.test.js b/src/validator/index.test.js
index b24d6d06308c0e17ece609644a7a899db4c10c8b..2091a177b70f0444c9a829ca57852084bf0c9e02 100644
--- a/src/validator/index.test.js
+++ b/src/validator/index.test.js
@@ -107,11 +107,19 @@ describe('Validator', () => {
     test('website should validate website format', () => {
       const validator = Validator.string().website('Invalid website')
 
-      addHttps.mockReturnValueOnce('https://invalid')
-      expect(validator.validate('invalid')).toBe('Invalid website')
+      addHttps.mockReturnValueOnce('http://')
+      expect(validator.validate('::')).toBe('Invalid website')
+
+      addHttps.mockReturnValueOnce('http://localhost')
+      expect(validator.validate('localhost')).toBeNull()
+
+      addHttps.mockReturnValueOnce('http://localhost:5173/en/login')
+      expect(validator.validate('http://localhost:5173/en/login')).toBeNull()
 
       addHttps.mockReturnValueOnce('https://example.com')
       expect(validator.validate('example.com')).toBeNull()
+
+      expect(validator.validate('')).toBeNull()
     })
 
     test('numeric should validate if string is numeric', () => {
